api-docs/docs/node-tracker/markdown/node-tracker.coreplugin.md

```diff
@@ -20,5 +20,6 @@ interface CorePlugin
 |  [afterTrack?](./node-tracker.coreplugin.aftertrack.md) | (payload: Payload) =&gt; void | <i>(Optional)</i> Called just after the trackerCore callback fires |
 |  [beforeTrack?](./node-tracker.coreplugin.beforetrack.md) | (payloadBuilder: PayloadBuilder) =&gt; void | <i>(Optional)</i> Called just before the trackerCore callback fires |

```

Comment:
I think we invalidate this now, it's no longer "just before" and might be called on events that are about to get filtered and never reach the callback?